platform/darwin/src/MGLRendererConfiguration.h

```diff
@@ -3,6 +3,16 @@
 
 NS_ASSUME_NONNULL_BEGIN
 
+/// Indicates how the map view load glyphs.
+typedef NS_ENUM(NSUInteger, MGLGlyphsRasterizationMode) {
+    /// No glyphs are rasterized locally. All glyphs are loaded from the server.
+    MGLNoGlyphsRasterizedLocally = 0,
+    /// Ideographs are rasterized locally, and they are not loaded from the server.
+    MGLIdeographsRasterizedLocally,
+    /// All glyphs are rasterized locally. No glyphs are loaded from the server.
+    MGLAllGlyphsRasterizedLocally
```

Comment:
These enums should have the same prefix, e.g. for Swift consumption. Although it's a mouthful, how about:

```
MGLGlyphsRasterizationModeNone
MGLGlyphsRasterizationModeIdeographsRasterizedLocally
MGLGlyphsRasterizationModeNoGlyphsRasterizedLocally
MGLGlyphsRasterizationModeAllGlyphsRasterizedLocally
```